<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_picture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1065</width>
      <height>671</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>670</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>Tsinghua University in Beijing</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>搜索</string>
    </property>
    <addaction name="action_search_student_info"/>
    <addaction name="action_teacher"/>
    <addaction name="action_depart"/>
    <addaction name="action_lesson"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>更多</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>问题联系</string>
     </property>
     <addaction name="action3463877866_qq_com"/>
    </widget>
    <addaction name="action"/>
    <addaction name="menu_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>学生信息</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>教师信息</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>课程信息</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>课程</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>学生</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>教师</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>系别</string>
   </property>
  </action>
  <action name="action_search_student_info">
   <property name="text">
    <string>学生信息</string>
   </property>
  </action>
  <action name="action_teacher">
   <property name="text">
    <string>教师信息</string>
   </property>
  </action>
  <action name="action_lesson">
   <property name="text">
    <string>课程信息</string>
   </property>
  </action>
  <action name="action_depart">
   <property name="text">
    <string>系别信息</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>学生成绩</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>教师评教分数</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>学生信息</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>教师信息</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>学生评教优秀课堂</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>查看源码</string>
   </property>
  </action>
  <action name="action3463877866_qq_com">
   <property name="text">
    <string>3463877866@qq.com</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
